src/scenes/ProgressReports/EditForm/Steps/Media/ImageField.tsx

```diff
@@ -88,15 +91,31 @@ export const ImageField = ({
         ...extendSx(props.sx),
       ]}
     >
-      <Box
-        component="img"
-        src={current.url}
-        crossOrigin="use-credentials"
-        sx={{
-          maxWidth: 1,
-          borderRadius: 1,
-        }}
-      />
+      {current.__typename === 'Image' ? (
+        <Box
+          component="img"
+          src={current.url}
+          crossOrigin="use-credentials"
+          sx={{
+            maxWidth: 1,
+            borderRadius: 1,
+          }}
+        />
+      ) : (
+        <Card
+          elevation={5}
+          sx={{
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+            maxWidth: 1,
+            borderRadius: 1,
+            aspectRatio: current.dimensions.aspectRatio,
+          }}
+        >
+          <Typography variant="button">Not an image</Typography>
```

Comment:
This isn't my favorite thing. Can we use another variant? Body1/2?

Reply:
Sure. I used button as it was upper case. I can use body1 instead if you'd like.